App.js

```diff
@@ -1,26 +1,88 @@
 import React from 'react'
 import styled from 'styled-components/native'
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 
-const Container = styled.View`
+import BookList from './BookList.js';
+import Details from './Details.js';
+
+//Style
+const Container = styled.ScrollView`
   flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
+  background-color: #fff;
 `
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
+const Body = styled.SafeAreaView`
+  flex: 1;
+  margin-top: 0;
 `
 
-const App = () => {
+//Home stack with fition and nonfiction books.
+const HomeStack = createStackNavigator();
+const HomeStackScreen = () => {
+ return (
+  <HomeStack.Navigator>
```

Comment:
Great job with Navigation :)

Nonetheless, all of those components should be moved to separate files